hcl/hcl.go

```diff
@@ -173,12 +187,12 @@ func Parse(path string) (HCL, error) {
 // BuildRunners steps through the HCLConfig structs and maps each runner config type to the corresponding New<Runner> function.
 // All custom runners are reduced into a linear slice of runners and served back up to the product.
 // No runners are returned if any config is invalid.
-func BuildRunners[T Blocks](config T, tmpDir string, c *client.APIClient, since, until time.Time, redactions []*redact.Redact) ([]runner.Runner, error) {
-	return BuildRunnersWithContext(context.Background(), config, tmpDir, c, since, until, redactions)
+func BuildRunners[T Blocks](config T, tmpDir string, debugDuration time.Duration, debugInterval time.Duration, c *client.APIClient, since, until time.Time, redactions []*redact.Redact) ([]runner.Runner, error) {
```

Comment:
These new runners need to use `debugDuration` and `debugInterval` as defaults. These values are usually passed down via `product.Config` but need to be individual params here to prevent `hcl` and `product` from mutually importing each other.

Reply:
Once we have specific runners for debug we ought to deprecate the debugDuration and debugInterval flags - their main usage was to pave over us not having an hcl-integrated runner.

Reply:
I'll create a ticket for that!